packages/lesswrong/lib/collections/revisions/schema.ts

```diff
@@ -154,7 +154,7 @@ const schema: SchemaType<DbRevision> = {
         // suggestion. Original contents is only visible to people who are invited 
         // to collaborative editing. (This is only relevant for posts, but supporting
         // it means we need originalContents to default to unviewable)
-        if (document.collectionName === "Posts") {
+        if (document.collectionName === "Posts" && document.documentId) {
```

Comment:
Actually, looking at the context here, I think this should be treated more like the `Revisions.checkAccess` logic.  Need to come back to this one and think about it a bit more.